default.nix

```diff
@@ -9,45 +9,76 @@ let
     inherit system;
   };
 
-  nix-dev = pkgs.stdenv.mkDerivation {
-    name = "nix-dev";
-    src = ./.;
-    nativeBuildInputs = with pkgs.python310.pkgs; [
-      linkify-it-py
-      myst-parser
-      sphinx
-      sphinx-book-theme
-      sphinx-copybutton
-      sphinx-design
-      sphinx-notfound-page
-      sphinx-sitemap
-    ];
-    buildPhase = ''
-      make html
-    '';
-    installPhase =
-      let
-        # Various versions of the Nix manuals, grep for (nix-manual)=
-        # FIXME: This requires human interaction to update!
-        # See ./CONTRIBUTING.md for details.
-        releases = [
-          "2.19"
-          "2.18"
-          "2.13"
-        ];
-        inputName = version: pkgs.lib.strings.replaceStrings [ "." ] [ "-" ] version;
-        src = version: (import inputs."nix_${inputName version}").default.doc;
-        copy = version: ''
-          cp -R ${src version}/share/doc/nix/manual/* $out/manual/nix/${version}
-        '';
-      in
-      with pkgs.lib.strings;
-      ''
-        mkdir -p $out/manual/nix/{${concatStringsSep "," releases}}
-        ${concatStringsSep "\n" (map copy releases)}
-        cp -R build/html/* $out/
+  nix-dev =
+    let
+      # Various versions of the Nix manuals, grep for (nix-manual)=
+      # FIXME: This requires human interaction to update!
+      # See ./CONTRIBUTING.md for details.
+      # Using Nix from Nixpkgs could simplify handling, but would requires a bit more setup here
+      releases = {
+        latest = "2.19";
+        rolling = "2.18";
```

Comment:
Is rolling the new unstable? If this is not the new accepted term, I would refrain from introducing it here.

Reply:
We started using "rolling" in the channel branch FAQ entry, and arguably it's the well-known term for what `nixpkgs-unstable` actually is. Also it's already used that way on the Nix manual overview page here, this just rewords it a bit.